lib/modules/manager/glasskube/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { Category } from '../../../constants';
+import { GlasskubePackagesDatasource } from '../../datasource/glasskube-packages';
+import * as GlasskubeVersioning from '../../versioning/glasskube';
+
+export { extractAllPackageFiles, extractPackageFile } from './extract';
+export const defaultConfig = {
+  versioning: GlasskubeVersioning.id,
+};
+export const categories: Category[] = ['kubernetes'];
```

Comment:
```suggestion
export const categories: Category[] = ['kubernetes', 'cd'];
```
Would you consider Glasskube a CD tool such as ArgoCD?
https://docs.renovatebot.com/modules/manager/

Reply:
Yeah, although it is not _only_ a CI platform, it can definitely be considered to fall into that category.